pkg/network/tracer/offsetguess/conntrack.go

```diff
@@ -319,6 +319,7 @@ func (c *conntrackOffsetGuesser) runOffsetGuessing(cfg *config.Config, ns netns.
 		// probe_kernel_read() handles faults gracefully.
 		if c.status.Offset_netns >= threshold || c.status.Offset_status >= threshold ||
 			c.status.Offset_origin >= threshold || c.status.Offset_reply >= threshold {
+			log.Debugf("Offset guessing overflowed at threshold %d and status %+v", threshold, c.status)
```

Comment:
This doesn't help us as we already know what threshold is, and the error message below outputs the status.

You should put some logs where we switch through cases for these offsets.

Reply:
this would log the entire status not just the name of the field. Not sure if we care about the additional info in `ConntrackStatus`.